package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-hss",
-  "version": "2.1.0",
+  "version": "3.1.0",
```

Comment:
My reasoning was that this constitutes a "breaking change" since projects that used to pass linting might now fail linting. A library should be able to lock to a major version only and not have their build broken by any new releases.